Zotero/Scenes/Detail/PDF/PDFCoordinator.swift

```diff
@@ -30,6 +30,17 @@ protocol PdfReaderCoordinatorDelegate: AnyObject {
 }
 
 protocol PdfAnnotationsCoordinatorDelegate: AnyObject {
+    func createShareAnnotationMenu(
```

Comment:
Based on our line width discussion I think these would fit single line? If they don't feel free to ignore this comment.

Reply:
Indeed, check https://github.com/zotero/zotero-ios/pull/693/commits/ee581b1572e8c7037696ffb6c6dd9f31ab72d30a